pipeline_components/_xgboost/Pipfile

```diff
@@ -10,6 +10,7 @@ kfp = "==1.8.9"
 pytest = ">=6.2.4,<7.0.0"
 xgboost = "==1.4.2"
 scikit-learn = "==0.24.1"
+joblib = "1.1.0"
```

Comment:
Does this need the "==" to be pinned? If it doesn't ignore me, not 100% on the syntax